src/bento-core/GlobalSearch/README.md

```diff
@@ -0,0 +1,28 @@
+# Introduction
+
+Global Search is a component that provides a implementation of a site-wide search. These components provide easy-to-use implementations for the Global Search feature. The two primary components provided by this package are:
+
+  1. SearchBar -- This bar is designed to be in the header throughout the application and is a typeahead (autocomplete) wrapper.
+  2. SearchResults -- This is the result section of a search, not to be confused with SearchBar results. It's found on the search page and has server-paginated search result cards (e.g. for Files, Programs, Samples, etc).
+
+# Importing
+
+```javascript
+import { SearchBarGenerator, SearchResults } from './GlobalSearch'; // Update path as needed
+
+// ... Use imports
+//
```

Comment:
@amattu2 Nice Documentation. I am curious about line 13 to line 20 in this file. Are these lines commented purposefully?

Reply:
@iksheth I don't think we need them, each component has better examples in the README. I've removed them and updated this README a little bit.